docs/HowToGuides/GettingStarted.md

```diff
@@ -187,7 +187,7 @@ Double-check that running `pwd` prints a path ending with `swift`.
 
 ### Spot check dependencies
 
-* Run `cmake --version`: This should be 3.18.1 or higher for macOS.
+* Run `cmake --version`: This should be 3.17 or higher.
```

Comment:
I'd like to point out something here. Based on my reading of [the CMake docs](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html) and usage over the years, the real de facto minimum after this pull will be 3.19.1, as that's what the CI will be enforcing. In that case, that's what should be listed as the minimum here in the docs.

@compnerd, can you confirm?

Reply:
One thing to be aware of is that the cmake issue you are talking about would only apply on Linux where the build behavior occurs. Windows/macOS on CI will just use what is on the system.

Reply:
Sure, the de facto minimum is the minimum CMake version used on the CI, ie `min(mac CI version, linux CI version, Win CI version)`. Raising this for the Linux CI will simply change one input to that equation.